MIT.md

```diff
@@ -1,10 +1,21 @@
-## Copyright 2021, [YOUR NAME]
+MIT License
 
-###### Please delete this line and the next one
-###### APP TYPE can be a webpage/website, a web app, a software and so on
+Copyright (c) [2023] [Abdessittir]
```

Comment:
- [ ] [OPTIONAL] Great job licensing this project 👍🏽 
Let's replace this field with your full-name 'Abdessittir Harkati'